LabeledIcon/LabeledIcon.js

```diff
@@ -77,10 +78,12 @@ const LabeledIconBase = kind({
  *
  * @hoc
  * @memberof moonstone/LabeledIcon
+ * @mixes moonstone/LabeledIcon.LabeledIconDecorator
```

Comment:
Could you replace `moonstone/LabeledIcon.LabeledIconDecorator` with `ui/LabeledIcon.LabeledIconDecorator`?

Reply:
😱